src/sql/azdata.proposed.d.ts

```diff
@@ -22,6 +22,14 @@ declare module 'azdata' {
 		export function openQueryDocument(options?: { content?: string; }, providerId?: string): Thenable<QueryDocument>;
 	}
 
+	/**
```

Comment:
1. The comment should be for the entire interface/class when put at that level. You should add a separate comment on the hasRowCount property and put the description for that there
2. It would still be good to modify azdata.d.ts directly and add the comment for IResultMessage (although something more descriptive than `Interface that represents a Result Message` would be good. What is a result message and where is it used? Look at the VS Code API for an example of good descriptive comments)